playground/tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+import pytest_container
+import os
+
+
+TW = pytest_container.Container(
```

Comment:
What is "TW" doing here?

Reply:
This is where pytest-container plugin runs the playground container.  TW is just a "name".

Reply:
I believe this is a remnant from one of the examples from `pytest_container`. The image I used in that example was "openSUSE Tumbleweed" which I just abbreviated with `TW`. But in the end it's just a name, albeit I think that `TW` is not very suitable here.

Reply:
Ah, now I understand what TW stands for. I was asking myself this question.
Replaced it with something which makes more sense to this use case.